src/sentry/models/sentryappavatar.py

```diff
@@ -26,7 +26,7 @@ class SentryAppAvatar(AvatarBase):
 
     FILE_TYPE = "avatar.file"
 
-    sentry_app = FlexibleForeignKey("sentry.SentryApp", related_name="avatar")
+    sentry_app = FlexibleForeignKey("sentry.SentryApp", related_name="avatars")
```

Comment:
You need to generate a migration to make this change, up to you but idk if it's worth it

Reply:
it is not 🤡